test/plugin/scenarios/okhttp-scenario/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+<Configuration status="debug">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout charset="UTF-8" pattern="[%d{yyyy-MM-dd HH:mm:ss:SSS}] [%p] - %l - %m%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <logger name="com.a.eye.skywalking.ui" level="debug" additivity="false">
+            <AppenderRef ref="Console"/>
+        </logger>
+        <!--<logger name="org.skywalking.apm.dependencies" level="off">-->
+        <!--</logger>-->
+        <Root level="debug">
+            <AppenderRef ref="Console"/>
+        </Root>
+    </Loggers>
+</Configuration>
```

Comment:
We redirect the `system.out` to `/dev/null`, so `system.out` is meaningless.

Reply:
ok

Reply:
I think you have a mistake. I mean to use [the other log configuration](https://github.com/apache/skywalking/blob/master/test/plugin/scenarios/grpc-scenario/src/main/resources/log4j2.xml) instead of it.

That is why we suggest using Archetype-Plugin.

Reply:
OK, i see.